sdk/cognitivelanguage/perf-tests/ai-language-text/test/languageDetection.spec.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfTest, PerfOptionDictionary, getEnvVar } from "@azure/test-utils-perf";
+import {
+  AzureKeyCredential,
+  TextAnalysisClient,
+  LanguageDetectionAction,
+  AnalyzeActionNames,
+} from "@azure/ai-language-text";
+import { DefaultAzureCredential } from "@azure/identity";
+
+interface LanguageDetectionPerfTestOptions extends LanguageDetectionAction {
+  "documents-count": number;
+}
+
+export class LanguageDetectionTest extends PerfTest<LanguageDetectionPerfTestOptions> {
+  options: PerfOptionDictionary<LanguageDetectionPerfTestOptions> = {
+    "documents-count": {
+      required: true,
+      description: "Number of documents",
+      shortName: "n",
+      longName: "docs-count",
+      defaultValue: 1000,
```

Comment:
does this actually test 1000 documents? I thought the service limit was lower.

Reply:
Limit for LD is 1k:
![image](https://user-images.githubusercontent.com/6074665/181298382-77b1fff7-63d5-4da5-94e3-979aa3e9bfc7.png)
See https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/concepts/data-limits